index.bs

```diff
@@ -1053,8 +1053,8 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 			<code>{ name: "clipboard-write", allowWithoutGesture: true }</code>
 			permission.
 
-		1. Let |hasGesture| be true if the current script is not running as a result of
-			direct user action, false otherwise.
+		1. Let |hasGesture| be true if the [=relevant global object=] of [=this=]
+		    has [=transient activation=], false otherwise.
 
 		1. If |writeWithoutGesture| is <a>granted</a>, then return true.
```

Comment:
This should be before the previous step, to correspond with the previous instance of this in lines 1020-1024

Reply:
For posterity, this was fixed in https://github.com/w3c/clipboard-apis/pull/122 (Thanks!)